src/dosbox.cpp

```diff
@@ -1192,27 +1193,40 @@ void DOSBOX_Init()
 	// DOS locale settings
 
 	secprop->AddInitFunction(&DOS_Locale_Init, changeable_at_runtime);
+
 	pstring = secprop->Add_string("locale_period", when_idle, "modern");
 	pstring->Set_help(
-	        "Set locale epoch ('modern' by default). Historic settings (if available\n"
-	        "for the given country) try to mimic old DOS behaviour when displaying\n"
-	        "information such as dates, time, or numbers, modern ones follow current day\n"
-	        "practices for user experience more consistent with typical host systems.");
-	pstring->Set_values({"historic", "modern"});
+	        "Set locale epoch ('native' by default).\n"
+	        "  historic:  if data is available for the given country, tries to mimic old DOS\n"
+	        "             behavior when displaying time, dates, or numbers\n"
+	        "  modern:    follow current day practices for user experience more consistent\n"
+	        "             with typical host systems\n"
+	        "  native:    tries to re-use current host OS settings, regardless of the country\n"
+	        "             set; uses 'modern' data to fill-in the gaps, like when the DOS\n"
+	        "             locale system is too limited to follow the desktop settings");
+	pstring->Set_values({"historic", "modern", "native"});
 
 	pstring = secprop->Add_string("country", when_idle, "auto");
 	pstring->Set_help(
 	        "Set DOS country code ('auto' by default).\n"
 	        "This affects country-specific information such as date, time, and decimal\n"
-	        "formats. The list of supported country codes can be displayed using\n"
-	        "'--list-countries' command-line argument. If set to 'auto', the country code\n"
-	        "corresponding to the selected keyboard layout will be used.");
+	        "formats. If set to 'auto', it tries to set the country code reflecting\n"
+	        "the host OS settings.\n"
+	        "The list of country codes can be displayed using '--list-countries'\n"
+	        "command-line argument.");
+
+	pstring = secprop->Add_string("keyboardlayout", deprecated, "");
+	pstring->Set_help("Renamed to 'keyboard_layout'.");
 
-	secprop->AddInitFunction(&DOS_KeyboardLayout_Init, changeable_at_runtime);
-	pstring = secprop->Add_string("keyboardlayout", when_idle, "auto");
+	pstring = secprop->Add_string("keyboard_layout", only_at_start, "");
 	pstring->Set_help(
-	        "Keyboard layout code ('auto' by default), i.e. 'us' for US English layout.\n"
-	        "Other possible values are the same as accepted by FreeDOS.");
+	        "Keyboard layout code (unset by default; this results in 'auto').\n"
```

Comment:
Hmm, should it be `auto` by default then? This is confusing.

Reply:
Indeed. Changed to 'auto'.